Framework/Kernel/src/FileValidator.cpp

```diff
@@ -90,6 +91,16 @@ std::string FileValidator::checkValidity(const std::string &value) const {
     return "File \"" + abspath + "\" not found";
   }
 
+  if (m_testExist && (Poco::File(value).exists())) {
+    std::ifstream in;
+    in.open(value.c_str());
+    if (!in) {
+      auto e = std::system_error(errno, std::generic_category(),
```

Comment:
Are we using `system_error` just to format the error message?

Reply:
Maybe there's a better way to do this, but I was using it to parse the system errror number into some error message that I could return

Reply:
Seems I can probably use `strerror` for this instead.. Ill change it